main/main.zkasm

```diff
@@ -15,6 +15,12 @@ start: ; main zkROM entry point
 ;;;;;;;;;;;;;;;;;;
         STEP => A
         0                                   :ASSERT ; Ensure it is the beginning of the execution
+
+        ; it is assumed that forkID is less than 32 bits
+        ; hence, 'JMPZ' instruction is safe
+        CTX                                 :MSTORE(forkID)
+        CTX - %FORK_ID                      :JMPNZ(failAssert)
```

Comment:
This can be expressed in just on line:
CTX - %FORK_ID => A                      :MSTORE(forkID), JMPNZ(failAssert)
Not sure if you have to set it to A or you can leave it like:
CTX - %FORK_ID.                             :MSTORE(forkID), JMPNZ(failAssert)